pkg/groovy/groovy_test.go

```diff
@@ -23,6 +23,7 @@ import (
 const configurationType = "test-conf-type"
```

Comment:
we need to fix these tests that we marked to be skipped, I will open an issue to address it